configuration/configuration_test.go

```diff
@@ -193,6 +195,22 @@ func TestGetKeycloakEndpointUserinfo(t *testing.T) {
 	assert.NotNil(t, configurationData.GetKeycloakEndpointUserinfo())
 }
 
+func TestMultipleConfigurations(t *testing.T) {
+	t.Parallel()
+
+	configurationData1, err := configuration.NewConfigurationData(tEnvironmentVariableValueConfigFile)
+	require.NotNil(t, configurationData1)
+	require.Nil(t, err)
```

Comment:
Please move the check for error before checking for `configurationData1`; otherwise in case of an error you would just see that `configurationData1` is `nil` but you wouldn't have a chance to find out why.